### ISSUE TITLE
Link Error in SSL.c (WOLFSSL) undefined reference to `LowResTimer' link error

### ISSUE THREAD

**Issue by tschesnok** [jondurrant/RPI-PicoW-WolfSSL-Exp#3]
Clean Git project. Latest FreeRTOS and WolfSSL in the /lib directory.

Everything builds nicely.. with one error:

make (2nd time to show error)
[  0%] Performing build step for 'PioasmBuild'
[100%] Built target pioasm
[  0%] No install step for 'PioasmBuild'
[  0%] Completed 'PioasmBuild'
[  1%] Built target PioasmBuild
[  1%] Built target cyw43_driver_picow_cyw43_bus_pio_spi_pio_h
[  2%] Built target bs2_default
[  2%] Built target bs2_default_padded_checksummed_asm
[  3%] Built target cyw43_firmware_package
[ 50%] Built target wolfssl
[ 50%] Performing build step for 'ELF2UF2Build'
[100%] Built target elf2uf2
[ 50%] No install step for 'ELF2UF2Build'
[ 50%] Completed 'ELF2UF2Build'
[ 51%] Built target ELF2UF2Build
[ 52%] Linking CXX executable HTTPSGet.elf
/usr/lib/gcc/arm-none-eabi/9.2.1/../../../arm-none-eabi/bin/ld: CMakeFiles/HTTPSGet.dir/home/andrew/Desktop/pico/Pico_ssl/lib/wolfssl/src/ssl.c.obj: in function `SetupSession':
ssl.c:(.text.SetupSession+0x6e): undefined reference to `LowResTimer'
collect2: error: ld returned 1 exit status
make[2]: *** [src/CMakeFiles/HTTPSGet.dir/build.make:3599: src/HTTPSGet.elf] Error 1
make[1]: *** [CMakeFiles/Makefile2:1860: src/CMakeFiles/HTTPSGet.dir/all] Error 2
make: *** [Makefile:152: all] Error 2
(base) andrew@andrew-ROG-Zephyrus-G15-GA503RM-GA503RM:~/Desktop/pico/Pico_ssl/HTTPSGet/build$ ls
CMakeCache.txt  cmake_install.cmake  CPackSourceConfig.cmake  FREERTOS_KERNEL  libwolfssl.a  pico-sdk  src
CMakeFiles      CPackConfig.cmake    elf2uf2                  generated        Makefile      pioasm

**Comment by tschesnok** [comment]
Oh man.. I'm never going to figure this out without help :(.  This must relate to the PICO port... something not working right with the time function definition.

**Comment by jondurrant** [comment]
I'll take a look. It looks like something has changed in WolfSSL. The instant fix would be to wind back to the version of WolfSSL I used for the testing.

**Comment by tschesnok** [comment]
Yes, Use HEAD 12e285434 of wolfssl as a work-around for now. Thanks Jon for helping on this.
